README: Merge event list readmes

edwarddavidbaker opened the pull request:
This pull request first copies the readme.txt files unmodified from https://download.01.org/perfmon/ . The content is then merged into the existing README.md file as a new section title Performance Monitoring Events. Badge icon markdown was also updated to comment out build (doesn't exist yet) and to fix the license badging.

https://github.com/intel/perfmon/issues/17

edwarddavidbaker commented:
@kshiprab Can you take a look when you get a minute? I mostly merged in the existing readme content from download.01.org. There are a few follow up items such as email addresses, and an updated notice section for later PRs, but wanted to get your initial take.

kshiprab reviewed (commented):
Hi Ed,
This looks good and well documented with comments having version numbers for files. 
Some of things I can think of
1. we need to copy tsv files. 
2. I think  you mentioned about email address and I think you will be updating this : perfmon-discuss@lists.01.org 
3. In Metrics section if we also have some documentation about TMA metrics excel files it will help user find the correct TMA file reference easily. 
4. The documentation on event lists needs to include uncore events as well. The documentation for offcore I think should be updated because offcore is merged into core files for new platforms from CLX onwards.
5. The whole section that contains this documentation needs to be reviewed carefully.
The event list filename indicates which type of list it contains, and follows this format:

`<microarchitecture-codename>_<core/offcore>_<version>`

edwarddavidbaker commented:
Thanks Kshipra!

I've added copying .tsv files and expanded TMA info to the list in https://github.com/intel/perfmon/issues/17. This pull request was also updated to mention uncore in the list of event type files.
